src/index.ts

```diff
@@ -1,7 +1,95 @@
+import { DataXY } from 'cheminfo-types';
+import sum from 'ml-array-sum';
+import variance from 'ml-array-variance';
+import covariance from 'ml-array-xy-covariance';
+import { EigenvalueDecomposition } from 'ml-matrix';
+
 /**
- * My module
- * @returns A very important number
+ * Calculates the Directional Ellipse for a set of points with a specific standard deviation
+ * @param {DataXY} [points]
+ * @param {number} [nbSD=2]
  */
-export function myModule(): number {
-  return 42;
+export function getDirectionalEllipse(points: DataXY, nbSD = 2) {
```

Comment:
Please use as second parameters `options` (an object) that contains currently only one property, `nbSD`